### ISSUE TITLE
Inaccurate Tuning ROC Curve generated at end of tuning

### ISSUE THREAD

**Issue by dvitale199** [GenoML/genoml#17]
<em>Please make sure that this is a bug.</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): RHEL 7.8 (biowulf)
- GenoML installed from (source or binary): Source
- GenoML version: python_v1.5
- Python version: 3.7

**Describe the current behavior**

Currently, a ROC curve is being generated for Tuning. Tuning ROC is, by design going to be pushed up to 1.0 because it is generated on training samples and is thus, not a useful output
<img width="959" alt="Screen Shot 2020-04-30 at 10 49 25 AM" src="https://user-images.githubusercontent.com/24816003/80814019-1f52d200-8b99-11ea-87a0-fb0d070676bc.png">

**Describe the expected behavior**
tuning should output table of mean, sd, min, and max for cv

**Code to reproduce the issue**
Run tuning and view output ROC curve

**Other info / logs**
None

**Comment by m-makarious** [comment]
GenoML previously only output the tuning metrics in the CLI, and exported the ROC curve as a `.png`

Now, the ROC is commented out in the code and is not automatically exported in favor of the following outputs are in `.csv` format.

**CSV Output no. 1:** Summary of the top 10 iterations of the hyperparameter tuning step + their parameters

This will now be generated with the suffix @ `*_tunedModel_top10Iterations_Summary.csv`

Example: 
![Screen Shot 2020-05-01 at 10 55 30 AM](https://user-images.githubusercontent.com/17086628/80819349-6d1c0a00-8b9a-11ea-8931-3ce5d6de2894.png)


**CSV Output no. 2:** Mean CV score + std dev + min CV score + max CV score at baseline vs. best tuned model

This will now be generated with the suffix @ `*.tunedModel_CV_Summary.csv`

Example:
![Screen Shot 2020-05-01 at 10 44 43 AM](https://user-images.githubusercontent.com/17086628/80818836-7b1d5b00-8b99-11ea-9b85-aae744fee07c.png)

In summary, when discrete or continuous tuning is run, these 2 `.csv` outputs will replace the ROC curve `.png ` for more useful outputs
